src/activeLighting.js

```diff
@@ -91,50 +91,50 @@ class ATL {
         Hooks.on("updateActiveEffect", async (effect, change, options, userId) => {
             if (game.userId !== userId) return;
             if (!effect.changes?.find(effect => effect.key.includes("ATL"))) return;
-            let totalEffects = effect.parent.effects.contents.filter(i => !i.disabled)
+            let totalEffects = effect.parent.actorEffects.contents.filter(i => !i.disabled)
```

Comment:
We can't just switch to `actorEffects` since that won't work for every other system. We'll need to do a check as to the system being used and either use `actor.actorEffects` if it's `wfrp4e` and `actor.effects` when it's not. I propose we add a small function to do that, then we can use it here:
```js
// I'd add this to the beginning of the `ready` function we're in
const getEffects = (actor) => {
    if (game.system.id === "wfrp4e") return actor?.actorEffects;
    return actor?.effects;
};
```
Then we can change lines like this to:
```suggestion
            let totalEffects = getEffects(effect.parent).contents.filter(i => !i.disabled)
```